File: shoppy-backend/src/main/java/com/app/shoppybackend/entity/Product.java
package com.app.shoppybackend.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "products")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Product {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    @Column(length = 1000)
    private String description;

    @Column(length = 2000)
    private String richDescription;

    private String category;
    
    private String subCategory;
    
    private String sizes;
    
    private Double price;
    
    private String imageUrl;
    
    @Column(length = 1000)
    private String images;
}

File: app/src/main/java/com/app/stylenest/model/Product.kt
package com.app.stylenest.model

data class Product(
    val id: Int,
    val name: String,
    val description: String,
    val richDescription: String,
    val category: String,
    val subCategory: String,
    val sizes: String,
    val price: Double,
    val imageUrl: String,
    val images: String // Comma-separated list for slideshow
)
